app/controllers/miq_policy_controller.rb

```diff
@@ -33,6 +33,11 @@ def edit
     miq_policy_edit
   end
 
+  def show
+    super
+    @expression_table = @record.expression.kind_of?(MiqExpression) ? exp_build_table(@record.expression.exp) : nil
```

Comment:
How was this implemented before? This seems too hacky for me :confused: :thinking:

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/blob/1b1760042b4127395898dd7b7d2c6799bab6fdc9/app/controllers/miq_policy_controller/policies.rb#L239-L257

Reply:
@skateman please re-review, added a helper method to get expression data into a table to use in view